src/App.tsx

```diff
@@ -1,61 +1,152 @@
+import original from 'react95/dist/themes/original';
```

Comment:
Consider importing only the necessary components from react95 instead of the entire library. This will reduce the bundle size and improve performance. 🚀